crates/bin/pd/src/info/oblivious.rs

```diff
@@ -263,6 +263,8 @@ impl ObliviousQueryService for Info {
                         .await??
                         .expect("compact block for in-range height must be present");
                     tx.send(Ok(block.to_proto())).await?;
+                    // Don't let the block hang around while we serve other clients.
+                    std::mem::drop(block);
```

Comment:
This isn't necessary, because the compact block doesn't have any shared references in it, and the explicit drop distracts from the case that is important (dropping the snapshot)